spiffworkflow-backend/src/spiffworkflow_backend/routes/process_instance_events_controller.py

```diff
@@ -164,3 +165,38 @@ def error_detail_show(
 
     error_details = process_instance_event.error_details[0]
     return make_response(jsonify(error_details), 200)
+
+
+def process_instance_migration_event_list(
+    modified_process_model_identifier: str,
+    process_instance_id: int,
+) -> flask.wrappers.Response:
+    # to make sure the process instance exists
+    process_instance = _find_process_instance_by_id_or_raise(process_instance_id)
+
+    logs = (
+        db.session.query(ProcessInstanceEventModel, ProcessInstanceMigrationDetailModel, UserModel)
+        .filter(
+            ProcessInstanceEventModel.process_instance_id == process_instance.id,
+            ProcessInstanceEventModel.event_type == ProcessInstanceEventType.process_instance_migrated.value,
+        )
+        .join(ProcessInstanceMigrationDetailModel)  # type: ignore
+        .outerjoin(UserModel, UserModel.id == ProcessInstanceEventModel.user_id)
+        .order_by(ProcessInstanceEventModel.timestamp.desc(), ProcessInstanceEventModel.id.desc())  # type: ignore
+        .add_columns(
+            ProcessInstanceEventModel.id,
+            ProcessInstanceEventModel.timestamp,
+            UserModel.username,
+            ProcessInstanceMigrationDetailModel.initial_bpmn_process_hash,
+            ProcessInstanceMigrationDetailModel.target_bpmn_process_hash,
+            ProcessInstanceMigrationDetailModel.initial_git_revision,
+            ProcessInstanceMigrationDetailModel.initial_git_revision,
+        )
+        .all()
+    )
+
+    response_json = {
+        "results": logs,
+    }
+
+    return make_response(jsonify(response_json), 200)
```

Comment:
we do not think we need pagination for this since it returns logs for only a single instance and migrations should not happen very often on a single instance.